src/logger/logger.cpp

```diff
@@ -83,91 +102,42 @@ void Logger::Init()
-
-        break;
-
-    case static_cast<int>(Module::eClockSync):
+    } else if (logModule == cLogModuleClockSync) {
         log_clocksync::LogCallback(level, message);
-
-        break;
-
-    case static_cast<int>(Module::eDownloader):
+    } else if (logModule == cLogModuleDownloader) {
         log_downloader::LogCallback(level, message);
-
-        break;
-
-    case static_cast<int>(Module::eOCISpec):
+    } else if (logModule == cLogModuleOCISpec) {
         log_ocispec::LogCallback(level, message);
-
-        break;
-
-    case static_cast<int>(Module::eProvisioning):
+    } else if (logModule == cLogModuleProvisioning) {
         log_provisioning::LogCallback(level, message);
-
-        break;
-
-    case static_cast<int>(Module::eResourceManager):
+    } else if (logModule == cLogModuleResourceManager) {
         log_resourcemanager::LogCallback(level, message);
-
-        break;
-
-    case static_cast<int>(Module::eRunner):
+    } else if (logModule == cLogModuleRunner) {
         log_runner::LogCallback(level, message);
-
-        break;
-
-    case static_cast<int>(Module::eStorage):
+    } else if (logModule == cLogModuleStorage) {
         log_storage::LogCallback(level, message);
-
-        break;
-
-        // Aos lib logs
-
-    case static_cast<int>(aos::LogModuleEnum::eCommonPKCS11):
-        log_pkcs11::LogCallback(level, message);
-
-        break;
-
-    case static_cast<int>(aos::LogModuleEnum::eCommonCrypto):
-        log_crypto::LogCallback(level, message);
-
-        break;
-
-    case static_cast<int>(aos::LogModuleEnum::eIAMCertHandler):
+    } else if (logModule == cLogModuleCerthandler) {
         log_certhandler::LogCallback(level, message);
-
-        break;
-
-    case static_cast<int>(aos::LogModuleEnum::eSMLauncher):
+    } else if (logModule == cLogModuleCrypto) {
+        log_crypto::LogCallback(level, message);
+    } else if (logModule == cLogModuleLauncher) {
         log_launcher::LogCallback(level, message);
-
-        break;
-
-    case static_cast<int>(aos::LogModuleEnum::eCommonMonitoring):
+    } else if (logModule == cLogModuleMonitoring) {
         log_monitoring::LogCallback(level, message);
-
-        break;
-
-    case static_cast<int>(aos::LogModuleEnum::eSMServiceManager):
+    } else if (logModule == cLogModuleServiceManager) {
         log_servicemanager::LogCallback(level, message);
-
-        break;
-
-    default:
-        LOG_MODULE_WRN(static_cast<aos::LogModule::EnumType>(Logger::Module::eCommunication))
-            << "Log from unknown module received: module = " << module << ", level = " << level
-            << ", message = " << message;
+    } else if (logModule == cLogModulePKCS11) {
+        log_pkcs11::LogCallback(level, message);
+    } else {
+        LOG_MODULE_WRN(cLogModuleCommunication.CStr()) << "Log from unknown module received: module = " << module
+                                                       << ", level = " << level << ", message = " << message;
```

Comment:
Remove spaces between var name and value:
`Log message: varName1=value1, varName2=value2`